ui/src/components/DigitalIdentityDetailPage.jsx

```diff
@@ -0,0 +1,70 @@
+import { Button, Input } from 'antd'
+import React from 'react'
+
+function DigitalIdentityDetailView() {
+    const [displayName, setDisplayName] = React.useState('')
+
+    const firstCardData = [
+        { name: "Claim name", value: "Base KYC", icon: false },
+        { name: "Claim expiration date", value: "30/07/2023", icon: false },
+        { name: "Issue date", value: "09/07/2023", icon: false },
+        { name: "Claim hash", value: "880bf326b5f662ba480ff48dc514631e", icon: true },
+        { name: "Txn hash", value: "0x5A55456454688474656454DDBVS", icon: true }
+    ]
+
+    const secondCardData = [
+        { name: "Claim name", value: "Accredited Investor", icon: false },
+        { name: "Claim expiration date", value: "30/07/2023", icon: false },
+        { name: "Issue date", value: "09/07/2023", icon: false },
+        { name: "Claim hash", value: "980bf326b5f662ba480ff48dc661188", icon: true },
+        { name: "Txn hash", value: "0x6A55456454688474656454DDRTQQA", icon: true }
+    ]
+
+    return (
+        <div >
+            <p className='text-xl p-6'>Identity Details</p>
+            <hr></hr>
+            <div className='p-6 max-[500px]:px-4 flex flex-col gap-4'>
+                <div>
+                    <p>Identity display name *</p>
+                    <div className='mt-3 relative w-full flex'>
+                        <Input className='w-full p-2 text-xl' placeholder='Base KYC' type='text' maxLength={32} onChange={(e) => setDisplayName(e.target.value)} />
+                        <p className='absolute right-5 top-2'>{displayName.length}/32</p>
+                    </div>
+                </div>
+                <div className='border rounded-xl p-6 bg-white flex flex-col gap-3 '>
+                    <p>Investor Wallet Address</p>
+                    <p className='font-light text-4xl max-[500px]:text-base text-gray-300'>0x5555aaadwerrasdw553</p>
+                    <p>Investor KYC ID Provider Account Number</p>
+                    <p className='font-light text-4xl max-[500px]:text-base text-gray-300'>007</p>
+                </div>
+                <p>Active Claims</p>
+                <VioletCard data={firstCardData} />
```

Comment:
Iterate through card data array here and create a card for each item

Reply:
Done